templates/main/singleton/boil_queries.go.tpl

```diff
@@ -12,10 +12,18 @@ var dialect = drivers.Dialect{
 	UseCaseWhenExistsClause: {{.Dialect.UseCaseWhenExistsClause}},
 }
 
+{{if not .AutoColumns.Deleted}}
+    // This is a dummy variable to prevent unused regexp import error
+    var _ = &regexp.Regexp{}
+{{- end}}
+
 // NewQuery initializes a new Query using the passed in QueryMods
 func NewQuery(mods ...qm.QueryMod) *queries.Query {
 	q := &queries.Query{}
 	queries.SetDialect(q, &dialect)
+	{{- if and (.AutoColumns.Deleted) (ne $.AutoColumns.Deleted "deleted_at") }}
+		queries.SetRemoveSoftDeleteRgx(regexp.MustCompile("{{$.AutoColumns.Deleted}}[\"'`]? is null"))
+	{{- end }}
```

Comment:
I think just using an init function would be better. This would call the `SetRemoveSoftDeleteRgx` every time a new query is created.

Reply:
Right! Changed :)